tests/EventSamplingTest.php

```diff
@@ -0,0 +1,291 @@
+<?php
+
+namespace Honeybadger\Tests;
+
+use Honeybadger\BulkEventDispatcher;
+use Honeybadger\Config;
+use Honeybadger\Honeybadger;
+use PHPUnit\Framework\TestCase;
+
+class EventSamplingTest extends TestCase
+{
+    /** @test */
+    public function it_samples_events_based_on_sample_rate()
```

Comment:
Nothing blocking, but I'd probably split this into `it_does_not_send_events_when_sample_rate_is_zero` and `it_sends_all_events_when_sample_rate_is_one_hundred`.